girder_worker/app.py

```diff
@@ -262,7 +262,7 @@ class _CeleryConfig:
 
 app = Celery(
     main=girder_worker.config.get('celery', 'app_main'),
-    backend=girder_worker.config.get('celery', 'broker'),
+    backend=girder_worker.config.get('celery', 'backend'),
```

Comment:
For backwards compatibility, should backend default to the broker setting if backend is not provided?